src/include/kdb.h.in

```diff
@@ -195,6 +203,21 @@ ssize_t keySetBinary(Key *key, const void *newBinary, size_t dataSize);
 int keyLock (Key * key, elektraLockFlags what);
 int keyIsLocked (const Key * key, elektraLockFlags what);
 
+static inline Key *keyDup (const Key *source)
```

Comment:
Also not needed.

Reply:
Sorry, I forgot to push a commit. The alias is acutally:

https://github.com/ElektraInitiative/libelektra/blob/49737e892bdef3922497c3b9f3caad0b7b2a274a/src/include/kdb.h.in#L206-L209

This allows duplicating only parts of `source`.